lib/internal/util.js

```diff
@@ -18,7 +18,11 @@ exports.printDeprecationMessage = function(msg, warned) {
 };
 
 exports.error = function(msg) {
-  console.error(`${prefix}${msg}`);
+  if (arguments.length > 1) {
+    arguments[0] = `${prefix}${msg}`;
```

Comment:
I checked and Crankshaft bails out with 'Bad value context for arguments value' as the deopt reason; TurboFan seems to be able to optimize it, though.

If writing it like this is unavoidable, I'd at least DRY the formatting into ```const fmt = `${prefix}${msg}`;```